docs/content_management/data_migration/importing_data.md

```diff
@@ -109,6 +109,76 @@ The expression syntax uses the following structure: `###<IDENTIFIER> <EXPRESSION
 
 The `IDENTIFIER` can be any repeated string that encloses the actual expression.
 
+#### Built-in functions
+
+Built-in expression language functions that are tagged with `ibexa.migrations.template.expression_language.function`:
+
+- `to_bool`, `to_int`, `to_float`, `to_string` - convert various data types by passing them into PHP casting functions (like `floatval`, `intval`, and others).
+
+```yaml
+                -   fieldDefIdentifier: show_children
+                    languageCode: eng-US
+                    value: '###XXX to_bool(i % 3) XXX###'
+
+                -   fieldDefIdentifier: quantity
+                    languageCode: eng-US
+                    value: '###XXX to_int("42") XXX###'
+
+                -   fieldDefIdentifier: price
+                    languageCode: eng-US
+                    value: '###XXX to_float("19.99") XXX###'
+
+                -   fieldDefIdentifier: description
+                    languageCode: eng-US
+                    value: '###XXX to_string(123) XXX###'
+```
```

Comment:
Can we remove redundant indentation from code block ?

Reply:
For this singular example it looks redundant, but in context of the whole page it is consistent with similar examples extracted from biger code snippets, and I would like it to stay that way.

<img width="901" alt="image" src="https://github.com/ibexa/documentation-developer/assets/35031941/47971d2f-d8d8-414e-8b40-2d400d9d1b27">